about.html

```diff
@@ -0,0 +1,138 @@
+                <p class="close">&times;</p>
+            </div>
+            <div class="information">
+                <ul>
+                    <li>
+                        <a href="#about" class="is-active">About</a>
+                    </li>
+                    <li>
+                        <a href="#" class="is-actives">Program</a>
+                    </li>
+                    <li>
+                        <a href="#" class="is-actives">Join</a>
+                    </li>
+                    <li>
+                        <a href="#" class="is-actives">Sponser</a>
+                    </li>
+                    <li>
+                        <a href="#" class="is-actives">News</a>
+                    </li>
+                </ul>
+            </div>
+            <div class="company">
+                <button>EEP Taining</button>
+            </div>
+        </div>
+    </nav>
+</header>
+    <div class="main">
+        <div class="hello">
+            <p class="heading">"Hello! Engineers in the world"</p>
+            <h2>Welcome to Electrical enginering <br> portal 2022</h2>
+            <p class="introduction">In this world nothing is possible without power.Power engineering, also called power systems engineering, is a subfield of electrical engineering that deals with the generation, transmission, distribution, and utilization of electric power, and the electrical apparatus connected to such systems.this seminar is a gate way  to power engineering.</p>
+            
+        </div>
+        <div class="contact">
+            <div class="contacts-desktop">
+                <p class="email">Please contact us per Email for any further questions</p>
+                <p class="eep">About EEP portal 2022!</p>
+            </div>
+            <div class="about">
+                <h2>desta451616@hotmail.com</h2>
+            </div>        
+        </div>
+        <section class="section3">
+        <div class="logo-seminar">
+            <div class="seminar-upper">
+                <h4>EEP portal</h4>
+            <h4 class="regional">regional seminar 2022</h4>
+            </div>
+            <p>the EEP portal seminar 2022 was decided <br> by Addis ababa institute of technology </p>
+            <img src="images/creative_commons.jpeg" alt="">
+            <hr>
+        </div>
+        
+            <div class="see-past">
+                <h3>see the past EEP regional seminar</h3>
+                <p>Take a look at the best two EEP Seminarswhich took place at in Addis Ababa and Mekele</p>
+            </div>
+            <div class="images-seminar">
+                <div class="seminar-2015">
+                    <h4>2015</h4>
+                    <p>EEP regional seminar 2015 in Addis Ababab</p>
+                </div>
+                <div class="seminar-2012">
+                    <h4>2012</h4>
+                    <p>EEP regional seminar 2015 in Mekele</p>
```

Comment:
- [ ]  Please align your content properly as requested by the previous reviewer. See below screenshots 👇
- Yours 👇 
![1](https://user-images.githubusercontent.com/67087664/181904825-a17d0100-4f51-474b-bb18-eb74cbd1565d.png)
- Original 👇 
![-1](https://user-images.githubusercontent.com/67087664/181904837-7b064ebd-0007-405e-8c7c-4fa50ef5a566.png)

Reply:
- [ ] Please align your content properly as requested by the previous reviewer. See below screenshots 👇
Yours 👇
![image](https://user-images.githubusercontent.com/63473584/182239010-12159e9e-31ad-419d-a5d5-465cf49a00ac.png)
Original 👇
![image](https://user-images.githubusercontent.com/63473584/182239201-eee68089-f7e1-49d1-a8ef-be97348e21d1.png)